### ISSUE TITLE
No support for Postgres 11 syntax with Zonky provider, unable to run docker inside docker

### ISSUE THREAD

**Issue by yuval-perelman** [zonkyio/embedded-database-spring-test#167]
We have a CI process using CircleCI that runs unit tests inside a docker container, and since there is no way to configure it to run a docker container inside a container we have to use the old Zonky provider.
(java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration).

The problem is that Zonky provider does not support Postgres 11 syntax. 

Any help would be appreciated.

**Comment by tomix26** [comment]
I'm not sure what you mean by the syntax not being supported. But I'm pretty sure the zonky provider supports postgres 11. I just checked it and here is my branch with a working example: https://github.com/tomix26/embedded-database-demo/tree/zonky-provider

Here are detailed instructions how to configure the provider correctly: https://github.com/zonkyio/embedded-database-spring-test#using-zonky-provider-previous-default

Let me know if that helped you.

**Comment by davidmchapman** [comment]
I can also confirm that the Zonky provider supports postgres 11 syntax. I came here thinking I had a similar issue as @yuval-perelman, but it turned out I was excluding the spring-context artifact from embedded-database-spring-test in my pom (not sure why I was doing that) and that was preventing the database upgrade from 10.15 to 11.11.

I solved the problem after following the link that @tomix26 provided above and comparing the demo pom to my own. Thank you!
